kore/src/Kore/ASTVerifier/SentenceVerifier.hs

```diff
@@ -437,3 +450,11 @@ parseAttributes'
     -> error attrs
 parseAttributes' =
     Attribute.Parser.liftParser . Attribute.Parser.parseAttributes
+
+parseAttributesAndVerify
```

Comment:
We should call this `parseAndVerifyAxiomAttributes`, because we will also be verifying other attributes soon.